pkg/sdn/plugin/node.go

```diff
@@ -204,9 +202,23 @@ func (node *OsdnNode) Start() error {
 		}
 	}
 
-	// Wait for kubelet to init the plugin so we get a knetwork.Host
-	log.V(5).Infof("Waiting for kubelet network plugin initialization")
-	<-node.kubeletInitReady
+	log.V(5).Infof("Waiting for kubelet and network plugin initialization")
+	err = kwait.ExponentialBackoff(
```

Comment:
if this returns a failure, the process exits, right? I'd rather poll without backoff at some reasonable interval (100-250ms) until systemd kills us. @smarterclayton, do you still prefer exponential backoff?